### DISCUSSION TITLE
Displayed time zone in the EJBCA Audit logs

### DISCUSSION THREAD

**Discussion by watdaf** [Keyfactor/ejbca-ce#452] [General]
While the EJBCA server and software is configured for PST, the audit logs display in UTC. Try as I might I cannot see of any where to change this display format to PST

![Greenshot 2023-12-21 09 37 12](https://github.com/Keyfactor/ejbca-ce/assets/102837272/5ac2c158-43fb-4e73-9a37-eb0200564523)

![Greenshot 2023-12-21 09 36 25](https://github.com/Keyfactor/ejbca-ce/assets/102837272/8f17a412-ec19-4205-a0b8-ed0bddc36600)

**Comment by primetomas** [comment]
For me it displays correct. It could/should be the servers timezone?
![Screenshot from 2024-01-16 14-32-16](https://github.com/Keyfactor/ejbca-ce/assets/28708578/57850951-ac00-47af-9bfb-840a5b957abc)
